docs/articles/user-guide/cli-commands.md

```diff
@@ -5,7 +5,7 @@ title: Run Salesforce CLI Commands
 To run a command from Salesforce Extensions for VS Code, press Ctrl+Shift+P (Windows or Linux) or Cmd+Shift+P (macOS) and type **SFDX** in the command palette.  
 ![Command palette, filtered to show SFDX commands](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-core/images/sfdx_commands.png)
 
-When a command finishes running (due to success, failure, or cancellation), a notification displays at the top of the window.  
+When a command finishes running (due to success, failure, or cancellation), a notification displays at the lower-right corner of the window.  
 ![Notification that source was successfully pushed to a scratch org](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-core/images/command_success_notification.png)
```

Comment:
This link is incorrect, it should be using images in `docs/images/`. There is an existing `docs/images/command_success_notification.png` file, I think we need to update that one instead of adding it to `packages/salesforcedx-vscode-core/images`

Reply:
Updated the image.